var/spack/repos/builtin/packages/hdf5-vol-daos/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Hdf5VolDaos(CMakePackage):
+    """The HDF5 DAOS VOL connector is an external VOL connector
+    that interfaces with the DAOS API"""
+
+    homepage = "https://github.com/HDFGroup/vol-daos"
+    url = (
+        "https://github.com/HDFGroup/vol-daos/releases/download/v1.2.0/hdf5_vol_daos-1.2.0.tar.bz2"
+    )
+    git = "https://github.com/HDFGroup/vol-daos.git"
+
+    maintainers("hyoklee", "soumagne")
+
+    version("master", branch="master", submodules=True)
+    version("1.2.0", sha256="669c1443605068f24c033783ef72619afcec4844902b3e0bffa19ddeea39779f")
+
+    depends_on("cmake@2.8.12.2:", type="build")
+    depends_on("daos@2.2.0:")
+    depends_on("hdf5@1.14.0:+hl+mpi+map")
```

Comment:
One audit check is complaining that `hdf5` does not have a `map` variant.

Reply:
https://github.com/spack/spack/pull/35523